src/main/java/com/minecolonies/coremod/commands/colonycommands/requestsystem/RSResetCommand.java

```diff
@@ -23,7 +23,7 @@
 public class RSResetCommand extends AbstractSingleCommand
 {
     public static final  String DESC            = "reset";
-    private static final String SUCCESS_MESSAGE = "After 1.618 Seconds it reinstantiated completely new.";
+    private static final String SUCCESS_MESSAGE = "After 1.618 seconds the colony was re-instantiated completely new.";
```

Comment:
Please revert this Change. It is an reference to something only a few People seem to know, yet I would still like to Keep it if it is possible :D

Reply:
I may have forgot about this for a few days, in this case would you also like the change in RSResetAll reverting too?

Reply:
Yeah